roadmap.md

```diff
@@ -2,184 +2,295 @@
-  </tr>
-  <tr>
-    <td>✅ Support non-transactional DML for insert and update operations.</td>
-    <td></td>
-  </tr>
-  <tr>
-    <td><ul><li>Implement a DDL parallel execution framework.</li><li>Provide DDL pause/resume capability.</li></ul></td>
-    <td>Implement a distributed parallel DDL execution framework, so that DDL tasks executed by only one TiDB Owner node can be coordinated and executed by all TiDB nodes in the cluster. Improve the execution speed of DDL tasks and cluster resource utilization.<br>By converting the execution of DDL tasks to distributed mode, this feature accelerates the execution speed of DDL tasks and improves the utilization of computing resources in the entire cluster. At present, DDL tasks that need to improve the speed include large table indexing and lossy column type modification tasks.</td>
-  </tr>
-  <tr>
-    <td rowspan="2">Hybrid Transactional and Analytical Processing (HTAP)</td>
-    <td>✅ Support TiFlash result write-back.</td>
-    <td><p>Support <code>INSERT INTO SELECT</code>.</p><ul><li>Easily write analysis results in TiFlash back to TiDB.</li><li>Provide complete ACID transactions, more convenient and reliable than general ETL solutions.</li><li>Set a hard limit on the threshold of intermediate result size, and report an error if the threshold is exceeded.</li><li>Support fully distributed transactions, and remove or relax the limit on the intermediate result size.</li></ul><p>These features combined enable a way to materialize intermediate results. The analysis results can be easily reused, which reduces unnecessary ad-hoc queries, improves the performance of BI and other applications (by pulling results directly) and reduces system load (by avoiding duplicated computation), thereby improving the overall data pipeline efficiency and reducing costs. It will make TiFlash an online service.</p></td>
-  </tr>
-  <tr>
-    <td>✅ Support FastScan for TiFlash.</td>
-    <td><ul><li>FastScan provides weak consistency but faster table scan capability.</li><li>Further optimize the join order, shuffle, and exchange algorithms to improve computing efficiency and boost performance for complex queries.</li><li>Add a fine-grained data sharding mechanism to optimize the <code>COUNT(DISTINCT)</code> function and high cardinality aggregation.</li></ul><p>This feature improves the basic computing capability of TiFlash, and optimizes the performance and reliability of the underlying algorithms of the columnar storage and MPP engine.</p></td>
-  </tr>
-  <tr>
-    <td>Maintenance</td>
-    <td>Support rule-based SQL blocklist.</td>
-    <td>In multi-service aggregation scenarios, provide SQL management and control capabilities, and improve cluster stability by prohibiting high-resource-consuming SQL statements.</td>
-  </tr>
-</tbody>
-</table>
-
-## Diagnosis and maintenance
+## Highlights of what we are planning
```

Comment:
Can we provide the last updated date in this roadmap to help anyone who read this to understand what "mid-year late" and "end of year late" means in time?

Reply:
@lucifinil PTAL

Reply:
If I understand correctly, the question has 2 parts:
1. what was the last updated date -> I assume it could be somehow equivalent to the last commit time ?
2. what does "mid-year stable" and "end of year stable" mean -> if the current is still not clear, we can do this: mid-year stable (targeting by June), End of year stable (targeting by Dec). but before the changes, just want to make sure it's not a misread, as I see "mid-year/end of year late" in the original question. 
Thanks!

Reply:
1. Yes, you are right. 
2. How about using "Mid-year plan, 1-year plan."? As for "stable", I am not sure if everyone will understand it the same.

Reply:
We could use the "Mid-year LTS release" and "End-of-year LTS release" instead in this case.

Reply:
Updated.